back/src/app.ts

```diff
@@ -1,5 +1,5 @@
 import { createApp } from './express.server';
-import { envConstants } from './env.constants';
+import { envConstants } from 'core';
```

Comment:
use here connectToDB. Group same imports

Reply:
Done and pushed